server/render.js

```diff
@@ -31,3 +31,39 @@ exports.renderReduxComponent = function renderReduxComponent(
     )
   );
 };
+
+/**
+ * This namespace a helper method for SSR using @loadable/component
+ * ```javascript
+ * import { ChunkExtractor } from "@loadable/server";
+ * import {renderReduxComponent} from "@quintype/framework/server/render";
+ * import path from "path";
+ *
+ * const statsFile = path.resolve("stats.json"); // This is the stats file generated by webpack loadable plugin
+ * const extractor = new ChunkExtractor({ statsFile, entrypoints: ["topbarCriticalCss", "navbarCriticalCss"] }); // We create an extractor from the statsFile
+ * renderLoadableReduxComponent(Header, params.store, extractor)
+ * The changes to be done in the app level for rendering critical Css will be updated shortly
+ * ```
+ * @category Server
+ * @module render
+ */
+/**
+ * @param {Component} Component The Component to render
+ * @param {Redux} store The store to render
+ * @param {extractor} props Extractor from the statsFile
+ * @param {Object} props The props to pass to the component
+ */
+exports.renderLoadableReduxComponent = function renderLoadableReduxComponent(
+  Component,
+  store,
+  extractor,
+  props
+) {
+  const children = React.createElement(Component, props);
+  //Wrap your component using "collectChunks"
+  const component = extractor.collectChunks(
```

Comment:
What happens when there are multiple components. Would it add stats and strings to the same chunk?

Reply:
Multiple chunks are created